admin3.php

```diff
@@ -22,45 +22,83 @@
+					<input style = "width: 18.8vw; height: 1.9vw; margin: 1vw 2vw 0vw 3.7vw; font-size: 1.2vw;" type = "text" name = "ExistingUserName" placeholder="Your Existing Username..."/>
+
+					<p style = "margin: 1.5vw 0vw 0vw 3.7vw; font-size: 1.4vw;">New Password:</p>
+					<input style = "width: 18.8vw; height: 1.9vw; margin: 1vw 2vw 0vw 3.7vw; font-size: 1.2vw;" type = "password" name = "NewPassword" placeholder="Your New Password..." id="myInput"/>
+
+					<p style = "margin: 1.3vw 0vw -1.3vw 5.3vw; font-size: 1vw;">Show Password</p>
+					
+					<!-- Checkbox for password visibility -->
+					<input type="checkbox" onclick="myFunction()"/>
+
+					<input style = "width: 19.2vw; height: 2.5vw; margin: 2.8vw 2vw 0vw 3.7vw; font-size: 1.2vw;" type = "submit" name = "submit" value = "Update Password"/>
 				
-				<!-- login box -->
-				<div class = "login_box">
+				</form></h3>
+
+				<center><p style = "margin: -5.8vw 0vw 0vw 0vw; font-size: 1vw; visibility: hidden;">Return Back To<a href = "login.php"> Login</a></p></center>
+
+			</div>
+
+			<!-- content_1 to hold header, and content_2 -->
+			<div class = "content_1">
+			
+				<!-- header class which consists of the header text -->
+				<div class = "header">
+					<b><p style = "color: black; font-size: 2vw; margin: 1vw 0vw 0vw 1vw; ">Header</p></b>
+				</div>
+
+				<!-- content_2, which contains content_3 -->
+				<div class = "content_2" style = "background-color: transparent; height: 17vw;">
 				
-					<b><p style = "color: black; font-size: 2vw; margin: 1vw 0vw 0vw 1vw;">Update Password</p></b>
-					
-					<h3><form method = "post" id= "01_login">
-						
-						<p style = "margin: 2.3vw 0vw 0vw 3.7vw; font-size: 1.4vw;">Existing Username:</p>
-						<input style = "width: 18.8vw; height: 1.9vw; margin: 1vw 2vw 0vw 3.7vw; font-size: 1.2vw;" type = "text" name = "username" placeholder="Your Existing Username..."/>
-						
-						<p style = "margin: 1.5vw 0vw 0vw 3.7vw; font-size: 1.4vw;">New Password:</p>
-						<input style = "width: 18.8vw; height: 1.9vw; margin: 1vw 2vw 0vw 3.7vw; font-size: 1.2vw;" type = "password" name = "password" placeholder="Your New Password..." id="myInput"/>
-						
-						<p style = "margin: 1.3vw 0vw -1.3vw 5.3vw; font-size: 1vw;">Show Password</p>
-						
-						<!-- Checkbox for password visibility -->
-						<input type="checkbox" onclick="myFunction()"/>
-						
-						<input style = "width: 19.2vw; height: 2.5vw; margin: 2.8vw 2vw 0vw 3.7vw; font-size: 1.2vw;" type = "submit" value = "Update Password"/>
-					
-					</form></h3>
-					
-					<center><p style = "margin: -6vw 0vw 0vw 0vw; font-size: 1vw; color: #ccb0f7;">Return Back To<a style = "color: #ccb0f7;" href = "LAYOUT_2 - login_v1.html"> Login</a></p></center>
+					<!-- content_3, which contains the query row title and query row output via SQL -->
+					<div class = "content_3" style = "background-color: transparent;">

```

Comment:
Added content_3, which contains the display users query title row and query output box.